bin/find-orphaned-files.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+#
+# This script checks the three core ownCloud documentation manuals to 
+# see if any of the example files are orphaned.
+#
+
+MANUALS=( admin developer user )
+
+echo "Checking for orphaned example files."
+echo
+
+for manual in "${MANUALS[@]}"; do
+    [[ ! -e "./modules/${manual}_manual/" ]] && echo "Cannot find ${manual} manual." && continue
+
+    echo "Checking the ${manual} manual"
+    find "./modules/${manual}_manual/examples/" \
+        \( -path "**/.DS_Store" -o -path "**/vendor" -o -path "**/.idea" -o -path "**/.gitkeep" \) -prune -o \
```

Comment:
`.idea` should never be part of some sub folder, so IMHO you could drop that.

Reply:
Generally correct, in this find statement, it is part of excluded directories to be searched in.
Whyever a directory named .idea is present, exclude it in the search.
Therefore it is ok to me.

Reply:
.idea folders are project folders by intellij, which would never be in a sub directory.

Reply:
fixed (removed)